packages/ckeditor5-code-block/src/codeblock.js

```diff
@@ -132,20 +132,23 @@ export default class CodeBlock extends Plugin {
  * The default value of the language configuration is as follows:
  *
  *		languages: [
- *			{ language: 'plaintext', label: 'Plain text' }, // The default language.
- *			{ language: 'c', label: 'C' },
- *			{ language: 'cs', label: 'C#' },
- *			{ language: 'cpp', label: 'C++' },
- *			{ language: 'css', label: 'CSS' },
- *			{ language: 'diff', label: 'Diff' },
- *			{ language: 'html', label: 'HTML' },
- *			{ language: 'java', label: 'Java' },
- *			{ language: 'javascript', label: 'JavaScript' },
- *			{ language: 'php', label: 'PHP' },
- *			{ language: 'python', label: 'Python' },
- *			{ language: 'ruby', label: 'Ruby' },
- *			{ language: 'typescript', label: 'TypeScript' },
- *			{ language: 'xml', label: 'XML' }
+*          { language: 'plaintext', label: 'Plain text' },
```

Comment:
```suggestion
*          { language: 'plaintext', label: 'Plain text' }, // The default language.
```

Let's restore the original comment - no reason to drop it.